proto/umee/oracle/v1/oracle.proto

```diff
@@ -111,3 +112,17 @@ message ExchangeRateTuple {
     (gogoproto.nullable)   = false
   ];
 }
+
+// AvgCounter is holds information about the active avg aggreator windows, used
+// to quickly compute average price of an asset.
+message AvgCounter {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string sum = 1 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  uint32                    num   = 2;
```

Comment:
nit: Maybe add a comment for what `num` represents here - or make the variable name more specific like `num_exchange_rates`

Reply:
OK, will add